src/redux.js

```diff
@@ -987,6 +987,7 @@ export function getDataDescription(state) {
 
 export function getDatasetDetails(state) {
   const datasetDetails = {};
+  datasetDetails.name = state.metadata.name;
```

Comment:
Oh, what happens for user-uploaded CSVs?  Do we want to save a name for them too?

Reply:
Great question. I asked @moneppo about it, and it's not a priority for this go-around. We'd like to know is the dataset user uploaded or not and if it is not, which of the in-house datasets was selected. Based on that metric (and curriculum feedback etc) if user uploaded datasets are common we can invest in having students name their datasets and maybe move some of the data collection questions currently on the model save form earlier in the flow to right after upload. For now, though, this is all we need :)